drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
we already have those methods.  that's what the fromFile method is calling. the only question is whether we should support the more generic "fromFile" method.  
i don't share your concerns for having that method.  in both cases, you might want to insert the model at a different location relative to the coordinate frame in the file (so I would like the frame argument to be supported in both loaders).  and I agree that sdf can have multiple robots while urdf has only one, but why is that a problem?